.github/workflows/rust.yml

```diff
@@ -86,6 +86,10 @@ jobs:
       - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
         with:
           ref: ${{ github.event.inputs.commit_sha }}
+      - uses: technote-space/get-diff-action@v4
```

Comment:
@elliotwutingfeng Instead of using an extra dependency. Would be better to just add another workflow, since github can trigger only on specific path changes:

https://github.com/build-trust/ockam/blob/develop/.github/workflows/rust.yml#L12-L32

Reply:
I set it to check for `**.toml` as well since changing it would usually change `Cargo.lock`.